book/src/feature/result.md

```diff
@@ -0,0 +1,31 @@
+# Result
```

Comment:
nit:

1. rename to `feature_result.md`
2. add a line in SUMMARY.md pointing to it